lib/PuppeteerSharp/Page.cs

```diff
@@ -726,46 +730,9 @@ public Task ExposeFunctionAsync<TResult>(string name, Func<TResult> puppeteerFun
         /// <returns>Task which resolves to the main resource response. In case of multiple redirects, the navigation will resolve with the response of the last redirect.</returns>
         /// <seealso cref="GoToAsync(string, int?, WaitUntilNavigation[])"/>
         public async Task<Response> GoToAsync(string url, NavigationOptions options)
```

Comment:
This could be
```cs
public Task<Response> GoToAsync(string url, NavigationOptions options)
    => _frameManager.MainFrame.GoToAsync(url, options);
```